join-2arrays-using-inbuilt.rb

```diff
@@ -0,0 +1,8 @@
+# Join 2 arrays using inbuilt functions.
+
+arr1 = ["Hello", "world"]
```

Comment:
Use `%w` syntax for array of strings.
https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#collections